californium-core/src/main/java/org/eclipse/californium/core/CoapServer.java

```diff
@@ -147,7 +147,7 @@ public CoapServer(final NetworkConfig config, final int... ports) {
 		}
 		
 		// resources
-		this.root = createRoot();
+		this.root = new RootResource();
```

Comment:
I didn't want to limit the possibilities, I was just suspecting that people might not want the default response with the `CoAP RFC Californium` bla bla bla ;-)